src/pages/resolve/util/blockchain.ts

```diff
@@ -0,0 +1,165 @@
+import { Arbitrator, SymbolInfo } from '../types';
+
+const VALID_CHARACTERS = 'abcdefghijklmnopqrstuvwxyz12345';
+
+export const validateId = (name: string) => {
+    const validCharacterList = VALID_CHARACTERS.split('');
+    let isValid = true;
+    if (name.includes('.')) {
+        if (name.length < 3 || name.length > 12) {
+            isValid = false;
+        }
+    } else {
+        if (name.length !== 12) {
+            isValid = false;
+        }
+    }
+    const nameCharacterList = name.split('');
+    for (const character of nameCharacterList) {
+        if (!validCharacterList.includes(character)) {
+            isValid = false;
+        }
+    }
+    return isValid;
+};
+
+export const generateRandomId = (length: number = 12) => {
+    let result = '';
+    const characters = 'abcdefghijklmnopqrstuvwxyz12345';
+    const charactersLength = characters.length;
+    for (let i = 0; i < length; i++) {
+        const char = characters.charAt(
+            Math.floor(Math.random() * charactersLength)
+        );
+        result += char;
+    }
+    return result;
+};
+
+export const validateIpfsHash = (url: string) => {
+    if (url.length !== 46 && url.length !== 49) return false;
+    return true;
+};
+
+export const fetchArbConfig = async (context: any) => {
+    const { rows } = await context.$store.$api.getTableRows({
+        code: process.env.ARB_CONTRACT,
+        scope: process.env.ARB_CONTRACT,
+        table: 'config'
+    });
+    const [config] = rows;
+    return config;
+};
+
+export const fetchArbitrators = async (context: any) => {
+    const { rows } = await context.$store.$api.getTableRows({
+        code: process.env.ARB_CONTRACT,
+        scope: process.env.ARB_CONTRACT,
+        table: 'arbitrators'
+    });
+    return rows;
+};
+
+export const fetchElections = async (context: any) => {
+    const { rows } = await context.$store.$api.getTableRows({
+        code: process.env.ARB_CONTRACT,
+        scope: process.env.ARB_CONTRACT,
+        table: 'elections',
+        reverse: true
+    });
+    return rows;
+};
+
+export const fetchNominees = async (context: any) => {
+    const { rows } = await context.$store.$api.getTableRows({
+        code: process.env.ARB_CONTRACT,
+        scope: process.env.ARB_CONTRACT,
+        table: 'nominees'
+    });
+    return rows;
+};
```

Comment:
above fetch methods are identical sans table name, can reuse the same method passing the table name